ledger/ledger-api-common/BUILD.bazel

```diff
@@ -21,6 +21,9 @@ da_scala_library(
         "@maven//:io_spray_spray_json",
         "@maven//:org_scalaz_scalaz_core",
     ],
+    scalacopts = [
+        "-Wconf:msg=class DeduplicationTime in object DeduplicationPeriod is deprecated:s",  # deprecated field that needs to be tested
```

Comment:
Can't we `nowarn` this closer to the actual reference?

Reply:
Also, [prefer `cat=deprecation`](https://twitter.com/stefanobaghino/status/1485539387806990338).

Reply:
That tweet seems wise. The author is clearly a brilliant individual.